src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,227 @@
+package duke;
+
+
+import java.util.ArrayList;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+public class Duke {
+    private static Parser parser;
+    private static UI ui;
+    private static Storage storage;
+    private static TaskList taskList;
+    public static void main(String[] args) throws IOException{
+        storage = new Storage();
+        ui = new UI();
+        parser = new Parser();
+        taskList = new TaskList();
+        String command = "";
+        Scanner in = new Scanner(System.in);
+        loadFile();
+        ui.showGreetings();
+        while (!command.equals("bye")) {
+            ui.printLinebreak();
+            command = in.nextLine();
+            String[] commandLine = parser.parseCommand(command);
+            ui.printLinebreak();
+            doCommand(commandLine);
+        }
+    }
+    private static void loadFile() throws IOException {
+        File file = storage.createFile();
+        Scanner fileReader = new Scanner(file);
+        while (fileReader.hasNext()) {
+            String[] savedDataLine;
+            savedDataLine = fileReader.nextLine().split("/");
+            if (savedDataLine[0].equals("[T]")) {
+                taskList.setStoredTasks(parser.parseTodo(savedDataLine, taskList.getStoredTasks()));
+            } else if (savedDataLine[0].equals("[D]")) {
+                taskList.setStoredTasks(parser.parseDeadline(savedDataLine, taskList.getStoredTasks()));
+            } else if (savedDataLine[0].equals("[E]")) {
+                taskList.setStoredTasks(parser.parseEvent(savedDataLine, taskList.getStoredTasks()));
+            }
+            taskList.incrementTaskNum();
+        }
+    }
+
+    private static void doCommand(String[] commandLine) {
+        switch (commandLine[0]) {
+        case "todo":
+            addTodoTask(commandLine);
+            break;
+        case "deadline":
+            addDeadlineTask(commandLine);
+            break;
+        case "event":
+            addEventTask(commandLine);
+            break;
+        case "mark":
+            markTask(commandLine);
+            break;
+        case "unmark":
+            unmarkTask(commandLine);
+            break;
+        case "list":
+            listTasks();
+            break;
+        case "delete":
+            deleteTask(commandLine);
+            break;
+        case "find":
+            findItem(commandLine);
+            break;
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
```

Comment:
could possibly use a message handler class to output this message instead of a println statement in the future!